calendar/endpoints.bal

```diff
@@ -264,6 +304,6 @@ public client class Client {
 # + oauth2Config - OAuth2 configuration
 # + secureSocketConfig- Secure socket configuration
 public type CalendarConfiguration record {
-    http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig oauth2Config;
+    http:BearerTokenConfig|http:OAuth2RefreshTokenGrantConfig|http:JwtIssuerConfig oauth2Config;
```

Comment:
@indikasampath2000 @abeykoon Is it okay to have the name `oauth2Config` for config? Changing name may break existing apps in Choreo.

Reply:
We need to add `@display` annotation to rendering a label instead field name in the record. Let's do it as a separate effort.

Reply:
Ack.